test/wait-task/README.md

```diff
@@ -0,0 +1,73 @@
+# Wait Custom Task for Tekton
+
+This repo provides an experimental [Tekton Custom
+Task](https://tekton.dev/docs/pipelines/runs/) that, when run, simply waits a
+given amount of time before succeeding, specified by an input parameter named
+`duration`.
+
+### Motivation
+
+Some users might want their Pipeline to wait some amount of time between two
+tasks, for example, to deploy to one cluster then wait an hour before deploying
+to other clusters.
+
+This is possible without this Custom Task using a Task that sleeps (e.g.,
+`script: sleep 600`), but this results in Pods running on the cluster that are
+just sleeping. This wastes cluster resources, and if nodes on the cluster go
+down then those sleeps will fail, leading to flaky deployments.
+
+Instead, this Custom Task centralizes all wait operations in the Custom Task
+controller, which is more efficient and more fault tolerant.
+
+This also acts as a simple Custom Task that can be modified to perform other
+more complex actions.
+
+## Install
+
+Install and configure `ko`.
+
+```
+ko apply -f controller.yaml
+```
+
+This will build and install the controller on your cluster, in the namespace
+`wait-task`.
```

Comment:
please update this readme, this custom task is meant for testing only -- not for users to install and use

Reply:
yeah that's true, thx for catching this, updated.